app/scripts/react-directives/view-svg-dashboard/pageStore.js

```
@@ -112,8 +107,15 @@ class ViewSvgDashboardPageStore {
   }
 
   switchValue(channel, value) {
-    let cell = this.deviceData.cell(channel);
-    cell.value = cell.value == value.on ? value.off : value.on;
+    if (!this.deviceData) {
+      return;
+    }
+    try {
+      let cell = this.deviceData.cell(channel);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'cell' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)